qiskit/providers/ibmq/api/session.py

```diff
@@ -259,3 +268,50 @@ def _modify_chained_exception_messages(self, exc: BaseException) -> None:
                 exc_message = exc_message.replace(self.access_token, '...')
             modified_args.append(exc_message)
         exc.args = tuple(modified_args)
+
+    def _log_request_info(
+            self,
+            url: str,
+            bare: bool,
+            method: str,
+            request_data: Dict[str, Any]
+    ) -> None:
+        """Log the request data, filtering out specific information.
+
+        Note:
+            The string ``...`` is used to denote information that has been filtered out
+            from the request, within the url and request data.
+
+            The following endpoint URLs are not logged in order to reduce noise: ``/users``
+            and ``/version``. Likewise, the request data is only logged for the following
```

Comment:
The docstring is now outdated with additional endpoints not being logged. It might be easier to either just say "not all endpoints are logged" or define a "constant" for excluded endpoints and just say those in the constant are not logged.

Reply:
I separated out filtering out url logs by introducing another private method [_is_worth_logging(...)](https://github.com/Qiskit/qiskit-ibmq-provider/pull/579/files#diff-fb7e1b8f723ef3e2f360841969f49284R315), in order to make filtering more readable and maintainable (in case we add other endpoints in the future). Would you rather have a "constant" be defined still or would the function be an okay addition instead?

Reply:
I personally like a "constant" since it's obvious and easy to update, but it's not a big deal to just use the method.